spec/models/warehouse/case_report_spec.rb

```diff
@@ -87,6 +87,7 @@
       create(:closed_case, :fully_refused_exempt_s40, :extended_for_pit),
       create(:sar_case, :extended_deadline_sar),
       create(:offender_sar_case, :third_party),
+      create(:offender_sar_case, :rejected),
```

Comment:
Spent a lot of time trying to add in tests for the rejected_reason however when testing the case_report output the tests seem very fiddly when dealing with the string formatting. E.g it changes the [] into "[]" and also also adds in escape chars ("/" and "\") when dealing with the rejected_reason string values. So left them out, the third party stuff doesnt seem to have tests either